func.php

```diff
@@ -0,0 +1,237 @@
+<link rel="stylesheet" href="styleoutput.css">
+<body>
+
+<?php
+session_start();
+class Base
+{
+    /** 
+    *
+    *Function to display the name of the user.
```

Comment:
@param can be used if function takes parameter.
it will be more easy to understand, if we comment function variable inside the function.